lib/_stream_transform.js

```diff
@@ -162,7 +162,7 @@ Transform.prototype._write = function(chunk, encoding, cb) {
 Transform.prototype._read = function(n) {
   var ts = this._transformState;
 
-  if (ts.writechunk !== null && ts.writecb && !ts.transforming) {
```

Comment:
26ca7d73 describes it as "null is reserved to indicate stream eof".  Perhaps the author confused `.write(null)` with `.push(null)`, or planned for `.write(null)` to cause EOF?  Or they intended to disallow `.write(null)` to avoid confusion about it being interpreted as EOF in other contexts?  I'm not exactly sure.